tracerite/html.py

```diff
@@ -20,7 +20,7 @@ def html_traceback(exc=None, chain=None, *, include_js_css=True, local_urls=Fals
             doc._style(style)
         for e in chain:
             if e is not chain[0]:
-                doc.p("The above exception occurred after catching", class_="after")
+                doc.p("The above exception occurred after catching the following (order left-to-right)", class_="after")
```

Comment:
Not a good place for this note (would only appear if there were multiple exceptions).

Maybe on the function call (arrow) tooltip something like `f"{current_func} called function {next_func}"`.

Or maybe on the tab bar:
* Minimalistic `f"⟶{final_symbol}"` after the last tab button, or
* Tab header buttons with icons (symbol of that frame)

Reply:
Trying to avoid adding a manual, such as a separate paragraph always at the start to explain it.

When proper suppressions are implemented, long call chains and multiple exceptions will be rare. Often all the code fits on screen, even if there are a couple of calls.

Reply:
Removed for now. We can figure this out later.